Containers/Camera/Camera.js

```diff
@@ -13,6 +13,8 @@ export class CameraWindow extends Component {
 
   componentWillMount = async () => {
     const { status } = await Permissions.askAsync(Permissions.CAMERA);
+    // const { status } = await Permissions.askAsync(Permissions.CAMERA_ROLL);
+    // console.log(status)
     this.setState({ hasCameraPermission: status === 'granted' });
   };
```

Comment:
Can we delete these comments?

Reply:
yes thanks, forgot I was going to get back to it. Pushed changes to HEAD